scripts/bash/run_enceladus.sh

```diff
@@ -412,6 +412,12 @@ get_temp_log_file() {
     mktemp -p "$LOG_DIR" -t "$TEMPLATE"
 }
 
+add_keytab_to_files() {
+    MENAS_AUTH_KEYTAB_NAME = echo "${MENAS_AUTH_KEYTAB}" | grep -o '[^/]*$'
```

Comment:
What this `grep` supposed to do? Seems to me, it takes the tail of the string ignoring front slashes. 🤔

Reply:
Sorry mixed up star and dot.